.github/workflows/msbuild.yml

```
@@ -80,21 +80,31 @@ jobs:
         md submodule/EplanIdleTimeModule/EplanIdleTimeModule/bin/${{env.BUILD_CONFIGURATION}}
         cp src/bin/${{env.BUILD_CONFIGURATION}}/* submodule/EplanIdleTimeModule/EplanIdleTimeModule/bin/${{env.BUILD_CONFIGURATION}}
 
-    - name: Build, test and analyze
+    - name: Sonar scanner begin
       working-directory: ${{env.GITHUB_WORKSPACE}}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         PULL_REQUEST_DATA: /d:sonar.pullrequest.key=${{ github.event.pull_request.number }} /d:sonar.pullrequest.branch=${{ github.event.pull_request.head.ref }} /d:sonar.scm.revision=${{ github.event.pull_request.head.sha }}
-      shell: powershell
-      run: |
-        .\.sonar\scanner\dotnet-sonarscanner begin /k:"savushkin-r-d_EasyEPLANner" /o:"savushkin-r-d" /d:sonar.login="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.cs.vscoveragexml.reportsPaths="D:\coverage.xml" ${{ github.event_name == 'pull_request_target' && env.PULL_REQUEST_DATA || '' }}  
-        msbuild -p:Configuration=${{env.BUILD_CONFIGURATION}} ${{env.SOLUTION_FILE_PATH}}
-        cd ./EasyEplanner.Tests
-        dotnet test --configuration ${{env.BUILD_CONFIGURATION}} --collect "Code Coverage"
-        dotnet-coverage merge -o D:\coverage.xml -f xml -r *.coverage
-        cd ..
-        .\.sonar\scanner\dotnet-sonarscanner end /d:sonar.login="${{ secrets.SONAR_TOKEN }}"
+      run: .\.sonar\scanner\dotnet-sonarscanner begin /k:"savushkin-r-d_EasyEPLANner" /o:"savushkin-r-d" /d:sonar.login="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.cs.vscoveragexml.reportsPaths="D:\coverage.xml" ${{ github.event_name == 'pull_request_target' && env.PULL_REQUEST_DATA || '' }}
+      
+    - name: Build
+      working-directory: ${{env.GITHUB_WORKSPACE}}
+      run: msbuild /m /p:Configuration=${{env.BUILD_CONFIGURATION}} ${{env.SOLUTION_FILE_PATH}}
+
+    - name: Test
+      working-directory: EasyEplanner.Tests
+      run: dotnet test --configuration ${{env.BUILD_CONFIGURATION}} --collect "Code Coverage"
+      
+    - name: Test coverage merge
+      working-directory: EasyEplanner.Tests
+      run: dotnet-coverage merge -o D:\coverage.xml -f xml -r *.coverage
```

Почему здесь такой путь к файлу с результатами (`D:\coverage.xml`)?
```suggestion
      run: dotnet-coverage merge -o D:\coverage.xml -f xml -r *.coverage
```